conf/server.yaml.template

```diff
@@ -13,6 +13,10 @@ SERVER:
     SOURCE: "internal"
     # The RHEL Base OS Version(x.y) where the Satellite is installed
     RHEL_VERSION: '7'
+  # If the the satellite server is IPv6 server
+  # IS_IPV6:
```

Comment:
@ogajduse Good question. but there would be cross testing in future as well like :
- ipv6 sat with ipv4 cap
- ipv6 sat with ipv4 host

I haven't thought about that design yet though since we are yet to plot that matrix. So I would keep this as is for now because all Ipv6 testing for now is being derived from server network type. We can always change the settings in future as design changes .